Specs/customizeJasmine.js

```diff
@@ -42,6 +42,8 @@ define([
             });
         }
 
+        window.specsUsingRelease = release;
```

Comment:
Can we just call this `window.release`?

Reply:
I think `release` is too simple of a variable name to set globally.